src/device-api/v1.ts

```diff
@@ -184,25 +182,13 @@ router.post('/v1/purge', (req: AuthorizedRequest, res, next) => {
 		.catch(next);
 });
 
-router.post('/v1/update', (req, res, next) => {
-	eventTracker.track('Update notification');
-	if (isReadyForUpdates()) {
```

Comment:
Removed the `isReadyForUpdates` check, any downsides of this?

Reply:
It seems like the target state module doesn't really depend on API binder so I think there should not be any issues. Still might be worth running this PR on a device and test it for a bit on real conditions.

Reply:
I will check this on a device tomorrow!

Reply:
I tested this by pinning to another release while simultaneously restarting the draft Supervisor. I haven't been able to see any issues, but there could still be a tiny window when `isReadyForUpdates` returns false and an update notification is received. Looking at the code, the window is mostly comprised of provisioning logic, so I also tested on a fresh device with the draft Supervisor preloaded, to see if changing the pinned release back & forth while the device still provisioning had any effect. I also didn't find any issues with device behavior, so I think this removal is fine.